umm-spec-lib/src/cmr/umm_spec/test/expected_conversion.clj

```diff
@@ -338,10 +338,10 @@
                                      :StartCircularLatitude 50.0
                                      :StartCircularLatitudeUnit "Degree"}}
    :TilingIdentificationSystems [{:TilingIdentificationSystemName "MISR"
-                                   :Coordinate1 {:MinimumValue 1.0
-                                                 :MaximumValue 10.0}
-                                   :Coordinate2 {:MinimumValue 1.0
-                                                 :MaximumValue 10.0}}]
+                                   :Coordinate1 {:MinimumValue "1.0"
+                                                 :MaximumValue "10.0"}
+                                   :Coordinate2 {:MinimumValue "1.0"
+                                                 :MaximumValue "10.0"}}]
```

Comment:
for this particular file expected_conversion.clj, i believe the purpose of this structure is to act as a 'base' that tests in other namespaces can modify as needed via assoc dissoc etc. since the vast majority of our records remain numeric strings here in this field, i thought the 'base' should reflect that for ease.